Automattic/wp-calypso
# Sites: remove default page opt-in banner

## Proposed Changes

This is a request from @pablohoneyhoney. He has complained that the banner is too prominent.

![image](https://github.com/Automattic/wp-calypso/assets/26530524/b3fb5c83-0a9e-4ad4-8c01-76d8ab034725)

The main idea behind this banner was to gauge interest in /sites, and understand how many people are willing to have this page as their default. The goal has been fulfilled, so it's time to remove this banner from the page itself and only control this setting through the Interface Settings card:

![image](https://github.com/Automattic/wp-calypso/assets/26530524/a8dbfe62-4f92-42dd-8a5f-d8e3a50c36ba)

## Testing Instructions

In an account with more than two sites, open `/sites` and verify that the banner is shown if you have not dismissed it within 14 days and do not have a preference already.

Do not dismiss it. Instead, open `/sites` with the same account but with this branch's code. You should not see the banner.
